### DISCUSSION TITLE
How to get time used with verifyta?

### DISCUSSION THREAD

**Discussion by Mars-bwu** [UPPAALModelChecker/UPPAAL-Meta#284] [Q&A] [answered]
I want to get time used with verifyta for every property, such as _Verification/kernel/elapsed time used: 1.187s / 0.016s / 1.235s_ 
Would someone please help me? Thank you very much.

**Comment by mikucionisaau** [comment, marked as answer]
Try `verifyta -u`
See help for more: `verifyta --help`

**Reply by Mars-bwu** [reply]
Thank you so much!!!
